### DISCUSSION TITLE
Device emulator for redmi note 10

### DISCUSSION THREAD

**Discussion by subhnehra007** [community/community#114294] [Discover: GitHub Best Practices]
I have rooted my device today (redmi note 10) I want to run device emulator to change imei number etc but it's not working. Please show me the way to download module with links . I will be grateful to you .

**Comment by queenofcorgis** [comment]
Hi @subhnehra007 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your device or the emulator. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
